src/app/examples/stock-market/components/stock-market-container.component.ts

```diff
@@ -18,14 +21,23 @@ export class StockMarketContainerComponent implements OnInit, OnDestroy {
   initialized;
   stocks;
 
-  constructor(public store: Store<State>) {}
+  constructor(
+    public store: Store<State>, 
+    public localStorageService: LocalStorageService) {
+
+      this.stocks  = localStorageService.getItem(STOCK_MARKET_KEY);
```

Comment:
Since we're using [NgRx](https://github.com/ngrx/platform) we don't want to call services directly from our components. If we want to retrieve something it has to come from the store:

```ts
this.store
      .pipe(select(selectStockMarket), takeUntil(this.unsubscribe$))
```

I haven't taken a detailed look yet, but @tomastrajan mentioned the following in the issue:

> probably is a race condition with initializing state from local storage / reducer initial state and expecting that state in the StockMarketComponent ngOnInit method.

Reply:
Oh ya, i forgot to put issue number. Ok then, i will try another solutions with ngrx way, thank you